### ISSUE TITLE
Support inherit policies when moving item

### ISSUE THREAD

**Issue by YanaDePauw** [DSpace/dspace-angular#333]
When moving an item, an option is present to make the item inherit the new collection’s policies. The checkbox is present in the UI, but no code is present yet to support the operation.

**Comment by tdonohue** [comment]
Yep, this is a duplicate of #1702, which was fixed in 7.6.  Closing.  See #1702 for more details
